ZenPacks/zenoss/ZenPackLib/lib/spec/GraphPointSpec.py

```diff
@@ -82,6 +82,7 @@ def __init__(
         # Allow color to be specified by color_index instead of directly. This is
         # useful when you want to keep the normal progression of colors, but need
         # to add some DONTDRAW graphpoints for calculations.
+        self.colorindex = colorindex
```

Comment:
i added this because repeated load/dump/load/dump cycles were failing when these didn't exist (there's another one too for RRDDatapointSpec 'shorthand')